### PR TITLE
[MB-7054] TIO can see pricing details for Domestic origin price

### PR DESCRIPTION
**Pull request by jenniferlynparsons**
## Description

Adds the Domestic origin pricing details

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-7054) for this change

## Screenshots

Screen Shot 2021-03-25 at 5 54 41 PM

### CODE REVIEW

File: src/components/Office/ServiceItemCalculations/helpers.js [hunk 1]
```diff
@@ -120,20 +151,29 @@ const makeCalculations = (itemCode, totalAmount, params) => {
...
       ];
       break;
+    case SERVICE_ITEM_CODES.DOP:
+      result = [
+        billableWeight(params),
+        originPrice(params),
+        priceEscalationFactor(params),
+        totalAmountRequested(totalAmount),
+      ];
+      // console.log(result);
```
Comment (donaldthai) [inline src/components/Office/ServiceItemCalculations/helpers.js:175]: Probably should remove this.

File: src/components/Office/ServiceItemCalculations/helpers.js [hunk 2]
```diff
@@ -53,6 +44,19 @@ const mileage = (params) => {
 
   return calculation(value, label, detail);
 };
+// const mileageZIP5 = (params) => {
```
Comment (donaldthai) [inline src/components/Office/ServiceItemCalculations/helpers.js:47]: Let's remove this too. @hannaliebl added the mileage zip 5 on their PR.

Comment (jenniferlynparsons) [reply src/components/Office/ServiceItemCalculations/helpers.js:47]: Cool! Away it goes.

**Review by donaldthai — APPROVED** [review]
LGTM. For those commented out tests, we can clean those up in the base branch.

**Pull request merged**